From 90c37e55fceae6e8fabf5da75ce37ed1c89c6631 Mon Sep 17 00:00:00 2001
From: Alistair Francis <alistair.francis@wdc.com>
Date: Wed, 17 Apr 2019 15:27:11 -0700
Subject: [PATCH] sifive-fu540: Set default arguments to help booting

Set some default args that will help us to autoboot. As the sifive-u540
currently does not support SD/MMC accesses we can't use the standard
uEnv.txt file to set extra variables.

This patch should be removed and replaced with a standard uEnv.txt flow
when u-boot can load defaults from the SD card.

Signed-off-by: Alistair Francis <alistair.francis@wdc.com>
Upstream-Status: Inappropriate [configuration]
---
 include/configs/sifive-fu540.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/sifive-fu540.h b/include/configs/sifive-fu540.h
index 7007b5f6af..07c6bfa178 100644
--- a/include/configs/sifive-fu540.h
+++ b/include/configs/sifive-fu540.h
@@ -38,6 +38,7 @@
 	"scriptaddr=0x82300000\0" \
 	"pxefile_addr_r=0x82400000\0" \
 	"ramdisk_addr_r=0x82500000\0" \
+	"serverip=@SERVERIP@\0" \
 	BOOTENV
 
 #endif /* __CONFIG_H */
-- 
2.21.0

